Userland/Libraries/LibELF/DynamicLoader.cpp

```diff
@@ -612,11 +579,7 @@ DynamicLoader::RelocationResult DynamicLoader::do_relocation(const ELF::DynamicO
         }
         break;
     }
-#if ARCH(I386)
-    case R_386_IRELATIVE: {
-#else
```

Comment:
This ifdef soup is quite sloppy, so I think it's ok to not keep it as arch(x86_64) and we should do a better job of keeping things separate for aarch64.

Reply:
OK, so I guess I should mark this as not resolved just to keep this as an open note?

Reply:
Hmm I think it's just a comment from me on the state of the file, so you can resolve